src/azure-cli/azure/cli/command_modules/appservice/custom.py

```diff
@@ -756,6 +756,11 @@ def update_auth_settings(cmd, resource_group_name, name, enabled=None, action=No
     return _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'update_auth_settings', slot, auth_settings)
 
 
+def list_instances(cmd, resource_group_name, name, slot=None):
+    # API Version 2019-08-01 does not return slot instances, however 2018-02-01 does
+    return _generic_site_operation(cmd.cli_ctx, resource_group_name, name, 'list_instance_identifiers', slot)
```

Comment:
Will _generic_site_operation call v2018-02-01 function here?

Reply:
Yes it will. Not being familiar with this codebase, I didn't know how to target a specific version. If you can tell me, I will update the code immediately and test it out.

Reply:
I misread your question. This function call will not target the 2018-02-01 API since I don't know how to call it. I'd love to target something that is known to work if you can tell me how.

Reply:
@Juliehzl

I found sample code from `get_appconfig_service_client` which showed how to target a specific API version. The underlying `get_mgmt_service_client` supported it, we just needed to add `api_version` to the stack of commands to get there. The default is 'None' which continues to use the latest version. This change will only impact this `list-instances` command.